BLSQ/iaso
# IA-2542: Permission "Submissions - Read and Write" only (without permission "Submissions - Read only") does not show in menu entry

Explain what problem this PR is resolving
- User with read and write submission permission only had not access to the submission menu
- The PR is about to let a user with read and write access the submission menu

Related JIRA tickets : [IA-2542](https://bluesquare.atlassian.net/browse/IA-2542)

## Self proofreading checklist


## Changes
- Added the SUBMISSION_UPDATE permission in instancePath

## How to test
- Update or create a user with submission read and write permission but without submission read only permission
- Check if the use has access to the submission on the sidebar menu

[IA-2542]: https://bluesquare.atlassian.net/browse/IA-2542?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
